_partials/_cloud-connect.md

```diff
@@ -1,67 +1,56 @@
+    In [Timescale Console][services-portal], check that your service is marked as `Running`.
 
-   ![Check service is running](https://assets.timescale.com/docs/images/cloud-sql-editors.png)
+   ![Check service is running](https://assets.timescale.com/docs/images/console-services-view.png)
 
 1. **Connect to your service**
 
-    Use either: 
-    - A Cloud SQL editor from Timescale Console:
-      - [SQL editor][run-sqleditor]: in Timescale Console, click `SQL editor`. You can now run queries for this service. 
-      - [PopSQL][popsql]:  in Timescale Console, click `PopSQL`, then follow the instructions to easily connect to 
-        this service with PopSQL.
-    - The command line:
-      - [psql][install-psql]: connect to your service with the value of `Service URL` from the config file you 
+    Use either:  
+    - [Data][popsql] in Timescale Console:  
+      1. In [`Data` in Timescale Console][portal-data-mode], select a service and enter your password, then click `Connect`.
+      
+         You find your password in the config file you just downloaded.
+      2. Select a query to edit:
+         ![Select a query to edit](https://assets.timescale.com/docs/images/data-mode-query-window.png)
+
+    - [SQL editor][run-sqleditor] in Timescale Console:
+    
+       In [`Ops` in Timescale Console][portal-ops-mode], select a service, then click `SQL editor`. 
+   
+      ![Check service is running](https://assets.timescale.com/docs/images/ops-view-sql-editor.png)
+   
+    - [psql][install-psql] on the command line:
+   
+      Connect to your service with the value of `Service URL` from the config file you 
         just saved:
 
         <CodeBlock canCopy={true} showLineNumbers={false} children={`
         psql "postgres://tsdbadmin:<PASSWORD>@<HOST>:<PORT>/tsdb?sslmode=require"
         `} />
 
-    You are connected to your service and can issue commands. 
+   You can now run queries for this service.  
 
-1. **Create a PostgreSQL table** 
 
-    Copy the following into your SQL editor, then run your query:
-
-    ```sql
-    CREATE TABLE stocks_real_time (
-      time TIMESTAMPTZ NOT NULL,
-      symbol TEXT NOT NULL,
-      price DOUBLE PRECISION NULL,
-      day_volume INT NULL
-    );
-    ```
-
-1.  **Check that the table exists**
-
-    - **SQL editor**: run the following query:
-      ```sql
-      SELECT * FROM information_schema.tables where table_name = 'stocks_real_time';
-      ```
-      You see the table listed in the UI.
-
-    - **PopSQL**: the table is automatically displayed when you run the `CREATE` query.  
-    - **psql**: run the `\dt` command. You see the table listing for your service. To disconnect, type `exit`.
-
-
-Quick recap, you find configuration information about your
-services in [Services section in Timescale Console][tsc-portal], you find configuration and security information in your
-config file.
+Quick recap. You:
+- Manage your services in the [`Ops` in Timescale Console][portal-ops-mode]
```

Comment:
Might be good to give some examples to drive the point home, like:
- Manage your services in `Ops mode` (eg add a replica, change parameters, etc)
- Analyze your data in `Data mode` (eg save queries, create charts/dashboards, etc)